cli/cmd/push/push.go

```diff
@@ -168,16 +161,14 @@ func buildApp(buildID string, appPath string) (ok bool) {
 		output.PrintErrorDetails("Error listening for build logs.", err)
 		return false
 	}
-
-	fmt.Println(greenCheckmark + "  Building app.......Done")
+	task.Done()
 
 	return true
 }
 
 func uploadApp(appDir string, toolID string) (buildID string, ok bool) {
 	defer os.Remove(zipFileName)
-
-	fmt.Print(unicodeHourglass + "  Uploading app......")
+	task := output.StartTask("Uploading app......")
```

Comment:
Isn't the "......" part of the `StartTask` function now?

Reply:
True!

Reply:
It does not make any difference in the output to remove it, but I will for clarity